### PR TITLE
Added load panel to indicate Dropzone is still working.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: assets/js/vue/FileManager.vue [hunk 1]
```diff
@@ -316,11 +328,14 @@ const downloadItems = (items) => {
     })
 }
 
+let fileManagerResolve = [];
+
 const uploadFileChunk = (fileData, uploadInfo, destinationDirectory) => {
     destinationDir = destinationDirectory.path;
     return new Promise((resolve, reject) => {
+        myFileManager.$parent.loadingVisible =  true;
```
Comment (praneethpr) [inline assets/js/vue/FileManager.vue:336]: Need to add this to upload File/Folder button too

**Review by fightingtexasaggie — APPROVED** [review]
tested, reviewed.

File: assets/js/vue/FileManager.vue [hunk 2]
```diff
@@ -373,8 +396,34 @@ const initDropzone = () => {
...
+    });
+
     myDropzone.on("queuecomplete", function () {
         myFileManager.instance.repaint();
+        totalFiles = 0;
+        doneFiles = 0;
+    });
+
+    myDropzone.on("totaluploadprogress", function (uploadProgress, totalBytes, totalBytesSent) {
+        if (uploadProgress == 100) {
```
Comment (praneethpr) [inline assets/js/vue/FileManager.vue:420]: Type comparison

File: assets/js/vue/FileManager.vue [hunk 3]
```diff
@@ -373,8 +396,34 @@ const initDropzone = () => {
...
+            fileManagerResolve.forEach(async function(fileResolve) {
```
Comment (praneethpr) [inline assets/js/vue/FileManager.vue:421]: Async function is not required here, because we don't use await

**Review by praneethpr — APPROVED** [review]

**Pull request merged**